plugins/modules/ndo_synce_interface_policy.py

```diff
@@ -0,0 +1,277 @@
+short_description: Manage syncE Interface Policies on Cisco Nexus Dashboard Orchestrator (NDO).
+description:
+- Manage syncE Interface Policies on Cisco Nexus Dashboard Orchestrator (NDO).
+- This module is only supported on ND v3.1 (NDO v4.3) and later.
+author:
+- Anvitha Jain (@anvjain)
+options:
+  template:
+    description:
+    - The name of the template.
+    - The template must be a fabric policy template.
+    type: str
+    required: true
+  inteface_policy:
+    description:
+    - The name of the syncE Interface Policy.
+    type: str
+    aliases: [ name ]
+  interface_policy_uuid:
+    description:
+    - The uuid of the syncE Interface Policy.
+    - This parameter is required when the O(interface_policy) needs to be updated.
+    type: str
+    aliases: [ uuid ]
+  description:
+    description:
+    - The description of the syncE Interface Policy.
+    type: str
+  admin_state:
+    description:
+    - The administrative state of the syncE Interface Policy.
+    - The default value is disabled.
+    type: str
+    choices: [ enabled, disabled ]
+    default: disabled
+  sync_state_msg:
+    description:
+    - The sync state message of the syncE Interface Policy.
+    - The default value is disabled.
+    type: str
+    choices: [ enabled, disabled ]
+    default: disabled
+  selection_input:
+    description:
+    - The selection input of the syncE Interface Policy.
+    - The default value is disabled.
+    type: str
+    choices: [ enabled, disabled ]
+    default: disabled
+  src_priority:
+    description:
+    - The source priority of the syncE Interface Policy.
+    - The value must be an integer between 1 and 254.
+    - The default value is 100.
+    type: int
+  wait_to_restore:
+    description:
+    - The wait to restore of the syncE Interface Policy.
+    - The value must be an integer between 0 and 12.
+    - The default value is 5.
+    type: int
+  state:
+    description:
+    - Use C(absent) for removing.
+    - Use C(query) for listing an object or multiple objects.
+    - Use C(present) for creating or updating.
+    type: str
+    choices: [ absent, query, present ]
+    default: query
+extends_documentation_fragment: cisco.mso.modules
+"""
+
+EXAMPLES = r"""
+- name: Create a new syncE interface policy
+  cisco.mso.ndo_synce_interface_policy:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    template: ansible_fabric_policy_template
+    interface_policy: ansible_test_interface_policy
+    admin_state: enabled
+    sync_state_msg: true
+    selection_input: true
+    src_priority: 100
+    wait_to_restore: 5
+    state: present
+
+- name: Query a syncE interface policy with template_name
+    cisco.mso.ndo_synce_interface_policy:
+        host: mso_host
+        username: admin
+        password: SomeSecretPassword
+        template: ansible_fabric_policy_template
+        interface_policy: ansible_test_interface_policy
+        state: query
+    register: query_one
+
```

Comment:
Can you add an example to query an object using `uuid`?

Reply:
added

Reply:
added